elibrary-core/source/src/controller/java/com/javafee/elibrary/core/startform/RegistrationEvent.java

```diff
@@ -49,29 +49,16 @@ public static void forceClearRegistrationEvenet() {
 	}
 
 	private static boolean checkRegistration(String login, String password, String peselNumber, Role role) {
-		boolean result = false;
 		UserAccount userAccount = (UserAccount) HibernateUtil.getSession().getNamedQuery("UserAccount.checkIfUserDataLoginExist")
 				.setParameter("login", login).uniqueResult();
-		switch (role) {
-			case WORKER_LIBRARIAN:
-			case CLIENT:
-				result = performCheck(userAccount, password);
-			case ADMIN:
-				break;
-			case WORKER_ACCOUNTANT:
-				break;
-			default:
-				break;
-		}
-
-		return result;
+		return performCheck(userAccount, password);
 	}
 
 	private static boolean performCheck(UserAccount userAccount, String password) {
 		boolean result = false;
 		if (userAccount != null)
 			Params.getInstance().add("ALREADY_REGISTERED", RegistrationFailureCause.ALREADY_REGISTERED);
-		else if (Common.checkPasswordStrength(password))
+		if (Common.checkPasswordStrength(password))
```

Comment:
Could u please make it shorter? 🚀 Please check if the ternary operator could be used here to simplify the code.

Reply:
I tried but there is a problem with method add(), it is void and if we want use here trenary operator this should returns boolean, so i think this change is not effective.